.github/workflows/main-test.yml

```diff
@@ -0,0 +1,47 @@
+# Runs acceptance tests on merge to main
+name: Acceptance Tests
+
+on:
+  push:
+    branches:
+      - "main"
+      - "akwirick/fix-testing-requirements" # TODO remove post-verification
```

Comment:
Instead of this approach, I'd suggest:

```
on:
  push:
    branches:
      - main
    ...
  pull_request:
    paths:
      - .github/workflows/main-test.yml
```

That way this file is self-testing any time it's changed.

Reply:
That's gonna fail in hard ways for third-party PRs but maybe that's ok?

Reply:
Why will it fail?

Reply:
Third party PRs don't get access to the token that we use in this file.   That's the whole point of the PR (to address #47 )

Reply:
Ah gotcha. So they'll fail if they modify this file -- is there a manual release button we can press, or would that just be a hard failure that would prevent any outside contributor from ever modifying this file?

If the latter, could we add a `workflow_dispatch` or `issue_comment` trigger to this job so that a Cortesian could trigger it manually? Or would that run into the same problem?

Reply:
The release happens when we add a versioned tag and that's a first-party action.   I think ideally we'd require a green build on that too but that's a follow on item for me.

Reply:
I'm not sure TBH on either of those.   Not familiar with whether or not Github would view those as first-party actions or not since the branch itself would still be on a remote fork.